packages/visualizations/src/components/Table/types.ts

```diff
@@ -95,14 +95,28 @@ export type GeoColumn = BaseColumn & {
     }>;
 };
 
+/**
+ * Render an interactive text that displays an image in a tooltip on tab/hover.
+ */
+export type ImageColumn = BaseColumn & {
+    dataFormat: typeof DATA_FORMAT.image;
+    options?: {
+        display?: (v: string) => string;
+        alt?: (v: string) => string;
```

Comment:
Do we need a callback for alt? I don't see alt being built from URL anyways. Mabybe from another column, but practical case will most likely be `() => 'my alt'`, or `() => someVarFromElseWhere`, no?

Reply:
Indeed not very useful.
Maybe when a cell has access to all values for a row we can expose an option that takes these values as args